flag/service/prometheus/prometheus.go

```diff
@@ -2,4 +2,10 @@ package prometheus
 
 type Prometheus struct {
 	BaseDomain string
+	Storage    PrometheusStorage
+}
+
+type PrometheusStorage struct {
+	CreatePVC string
+	Size      string
```

Comment:
Having the whole storage configuration in a different PR would have been nice.

Reply:
It is in another one. I accidently used the wrong starting branch

Reply:
I will clean it up